### PR TITLE
Authentication: Login page

### PR DESCRIPTION
**Pull request by Gautam2305**
- added fixed navbar to be displayed in every page
- buttons for categories in nav bar
- login page with email & password inputs
- solid primary button to continue with login
- added signup pages 
- added logout page 

Screenshot (114)

Screenshot (115)

Screenshot (116)

### CODE REVIEW

**Review by rahulcs754 — APPROVED** [review]
i looked your code that clean and good and you explain every CSS in utilities class that is nice

File: pages/authentication_page/login.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    background-color: var(--st-light-pink);
+    max-width: 100%;
+    height: calc(100vh - 80px);
+}
+.brand-name{
+    width: 8rem;
+    color: var(--st-primary);
+}
+.nav-category{
+    display: flex;
```
Comment (aishwarya-bm) [inline pages/authentication_page/login.css:18]: overall code looks very clean and crisp, just one suggestion you can use util class for this

File: pages/authentication_page/login.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+}
+.category-link{
+    text-decoration: none;
+    padding: 0.65rem 1rem;
+    color: var(--st-primary-text);
+}
+.nav-icons{
+    display: flex;
```
Comment (aishwarya-bm) [inline pages/authentication_page/login.css:26]: same here, util can replace this
use if you have created them

**Review by aishwarya-bm — COMMENTED** [review]
For flex we can create some utils for centered display & column display & use them across

File: pages/authentication_page/login.css (hunk 1 above)
Comment (Gautam2305) [reply pages/authentication_page/login.css:18]: yeah i will use that later on ...did this due to time crunch

**Review by aishwarya-bm — APPROVED** [review]
code seems clean, you can go ahead.
you can add the utils once you get time

**Pull request merged**